models/migrations/v1_19/v239.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_19 //nolint
+
+import (
+	auth_models "code.gitea.io/gitea/models/auth"
+
+	"xorm.io/xorm"
+)
+
+func AddScopeForAccessTokens(x *xorm.Engine) error {
+	type AccessToken struct {
+		Scope auth_models.AccessTokenScope
+	}
+
+	if err := x.Sync(new(AccessToken)); err != nil {
+		return err
+	}
+
+	// all previous tokens have `all` and `sudo` scopes
+	_, err := x.Exec("UPDATE access_token SET scope = ? WHERE scope IS NULL OR scope = ''", auth_models.AccessTokenScopeAll+","+auth_models.AccessTokenScopeSudo)
+	return err
+}
```

Comment:
```suggestion
import (
	"xorm.io/xorm"
)

func AddScopeForAccessTokens(x *xorm.Engine) error {
	type AccessToken struct {
		Scope string
	}

	if err := x.Sync(new(AccessToken)); err != nil {
		return err
	}

	// all previous tokens have `all` and `sudo` scopes
	_, err := x.Exec("UPDATE access_token SET scope = ? WHERE scope IS NULL OR scope = ''", "all,sudo")
	return err
}
```

Just use `string` instead of `auth_models.AccessTokenScope` in migrations.

We may drop this feature and remove `auth_models.AccessTokenScope` in the future (don't be sad, I'm just assuming), but we still need `migrations` to compile properly. That's why we define `type AccessToken struct` here instead of using `auth_models.AccessToken`.